peter-test.js

```diff
@@ -0,0 +1,4 @@
+const toRegexRange = require('./');
+var source = toRegexRange('-128', '127', {capture: true, antlr: true});
+console.log("-128 -> 127 = "+source);
```

Comment:
Unit tests should use assertions. Please see the `test/test.js` file for examples of how tests are done.  I would be happy to help if you have questions.